README.md

```diff
@@ -1,17 +1,21 @@
 # `UI::HtmlDialog` Examples
 
-These examples were presented at SketchUp DevCamp 2017 in Leeds, UK.
+These examples were originally presented at SketchUp DevCamp 2017 in Leeds, UK.
 
-It is split into three parts:
-* HtmlDialog class (key difference from WebDialog).
-* Communication between Ruby and JS. HTML content synchronization.
-* Styling.
+They have later been updated and tweaked.
+
+The examples are grouped into three parts:
+* [HtmlDialog class](#part-1-htmldialog-class) (key difference from WebDialog).
+* [Communication between Ruby and JS.](#part-2-communication-between-ruby-and-js) HTML content synchronization.
+* [Styling.](#part-3-styling)
 
 ![](Screenshot.png)
```

Comment:
From the screenshot there seems to be rather much excess space between the dialog content an the footer buttons. Maybe shrink that a bit?

Reply:
Fixed height and updated image.